components/director/internal/tenantfetchersvc/provisioner.go

```diff
@@ -58,6 +94,10 @@ func (p *provisioner) ensureParentExists(ctx context.Context, parentTenantID, ch
 		return id, nil
 	}
 
+	if !automaticParentCreationEnabled(childTenant) {
```

Comment:
Why automatic GA creation as part of SA event is disabled? What will happen if we miss a subscription event for a subaccount because the GA is still not fetched?